techmatters/flex-plugins
# CHI-1183: e2e headless login

Requires Playwright POC (#593) to be merged first

Primary reviewer: @murilovmachado 

## Description

* Logs in via Okta SSO by directly calling the SAML APIs with a HTTP client rather than using the GUI in a browser session. Cuts setup time from >16 seconds to <5 seconds in local testing

### Checklist

### Related Issues
CHI-1183

### Verification steps

Run e2e tests, they should still pass. No browser should launch prior to executing the actual test.
